appservice/src/index.ts

```diff
@@ -36,5 +39,5 @@ export * from './tree/DeploymentsTreeItem';
 export * from './tree/DeploymentTreeItem';
 export * from './tree/ISiteTreeRoot';
 export * from './TunnelProxy';
-export { registerAppServiceExtensionVariables } from './extensionVariables';
-export { IConnectToGitHubWizardContext } from './github/IConnectToGitHubWizardContext';
+
```

Comment:
source.organizeImports is really annoying for files like these, but I like it for other files. Anyways only actual change is adding setLocationsTask